addons/web/static/tests/webclient/actions/client_action_tests.js

```diff
@@ -410,6 +410,51 @@ QUnit.module("ActionManager", (hooks) => {
         );
     });
 
+    QUnit.test("test display_notification client action with links", async function (assert) {
+        assert.expect(6);
+        const webClient = await createWebClient({ serverData });
+        await doAction(webClient, 1);
+        assert.containsOnce(webClient, ".o_kanban_view");
+        await doAction(webClient, {
+            type: "ir.actions.client",
+            tag: "display_notification",
+            params: {
+                title: "title",
+                message: "message %s",
+                sticky: true,
+                links: [{
+                    label: "test",
```

Comment:
could you test the escaping is correctly done (e.g. `label: "<R&D>"`) and that the link is correctly constructed?

Reply:
FWIW the display_notification stuff escapes everything internally, it was built on the basis of not trusting the server.